openapi/api.yml

```diff
@@ -121,7 +121,7 @@ paths:
                   description: JSON string; restrict ingest to submissions based on some fields in the data file. If
                     this is omitted then authorisation checks will not be run.
                     e.g. Towns Fund checks against Place Name and Fund Type.
-                  default: "{\"Place Names\": [\"Blackfriars - Northern City Centre\"], \"Fund Types\": [\"Town_Deal\", \"Future_High_Street_Fund\"]}"
+                  default: "{\"Programme\": [\"Blackfriars - Northern City Centre\"], \"Fund Types\": [\"Town_Deal\", \"Future_High_Street_Fund\"]}"
```

Comment:
It might be misleading if we change this before modifying TF's auth keys - probs best to revert these for now as well